src/AltinnCore/Common/Services/Implementation/ExecutionSIContainer.cs

```diff
@@ -0,0 +1,272 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using AltinnCore.Common.Backend;
+using AltinnCore.Common.Configuration;
+using AltinnCore.Common.Constants;
+using AltinnCore.Common.Services.Interfaces;
+using AltinnCore.ServiceLibrary;
+using AltinnCore.ServiceLibrary.Configuration;
+using AltinnCore.ServiceLibrary.ServiceMetadata;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.AspNetCore.Mvc.Razor.Compilation;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace AltinnCore.Common.Services.Implementation
+{
+  using AltinnCore.Common.Helpers;
+  using AltinnCore.Common.Helpers.Extensions;
+  using Microsoft.AspNetCore.Http;
+  using System.Reflection;
+  using System.Runtime.Loader;
+  using System.Text;
+
+  /// <summary>
+  /// Service that handle functionality needed for executing a Altinn Core Service (Functional term)
+  /// </summary>
+  public class ExecutionSIContainer : IExecution
+  {
+    private const string SERVICE_IMPLEMENTATION = "AltinnCoreServiceImpl.{0}.{1}_{2}.ServiceImplementation";
+
+    private readonly ServiceRepositorySettings _settings;
+    private readonly IRepository _repository;
+    private readonly Interfaces.ICompilation _compilation;
+    private readonly IViewRepository _viewRepository;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+    private Dictionary<string, string> _assemblyNames = new Dictionary<string, string>();
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ExecutionSILocalDev"/> class 
+    /// </summary>
+    /// <param name="settings">The repository setting service needed (set in startup.cs)</param>
+    /// <param name="repositoryService">The repository service needed (set in startup.cs)</param>
+    /// <param name="compilationService">The service compilation service needed (set in startup.cs)</param>
+    /// <param name="partManager">The part manager</param>
+    /// <param name="viewRepository">The view Repository</param>
+    public ExecutionSIContainer(
+        IOptions<ServiceRepositorySettings> settings,
+        IRepository repositoryService,
+        Interfaces.ICompilation compilationService,
+        ApplicationPartManager partManager,
```

Comment:
Can this be removed since its not used?

Reply:
You forgot to remove the param in the summary tab, but this can be fixed when we add stylecop